types/node/crypto.d.ts

```diff
@@ -2475,6 +2481,10 @@ declare module "crypto" {
          * Name of the curve to use
          */
         namedCurve: string;
+        /**
+         * Must be `'named'` or `'explicit'`. Default: `'named'`.
+         */
+        paramEncoding?: "explicit" | "named";
```

Comment:
I don't see this property in doc. Maybe you want to add this to `interface RSAKeyPairOptions`?

Reply:
According to the [Docs](https://nodejs.org/dist/latest-v21.x/docs/api/crypto.html#cryptogeneratekeypairsynctype-options) it is related to `EC` keys only, not `RSA`

> paramEncoding: [<string>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type) Must be 'named' or 'explicit' (EC). Default: 'named'.

Reply:
Ah, I see, you added to `interface ECKeyPairOptions`.